tests/testmediaqueries.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @fileoverview Defines tests for testMediaQueries.
+ */
+
+
+/**
+ * @return {boolean} return true if test failed
+ */
+function testMediaQueries() {
+  /** @const {!RegExp} */ var MEDIA_PATTERN = /@media([^{]*)/;
+  /** @type {string} */ var node = 'media-widget';
+  /** @type {Element} */ var element = document.getElementById(node);
+  /** @type {Element} */
+  var failClass = document.getElementsByClassName('rule media-none');
+  /** @type {Node} */ var table = document.getElementById('media-data-table');
```

Comment:
Few lines above:
```javascript
/** @type {Element} */ var element = document.getElementById
```
This line:
```javascript
/** @type {Node} */ var table = document.getElementById
```

> Returns a reference to an [Element](https://developer.mozilla.org/en-US/docs/Web/API/Element) object, or `null` if an element with the specified ID is not in the document.

https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById